templates/commands/render.go

```diff
@@ -322,3 +575,13 @@ func destOK(fs fs.StatFS, dest string) error {
 
 	return nil
 }
+
+func safeRelPath(p string) error {
```

Comment:
I'd like to see some unit tests for this please.

Reply:
Added tests.